clients/admin-ui/src/features/privacy-requests/privacy-requests.slice.ts

```diff
@@ -12,14 +12,15 @@ import {
 } from './types';
 
 // Helpers
-export const mapFiltersToSearchParams = ({
+export function mapFiltersToSearchParams({
   status,
   id,
   from,
   to,
   page,
   size,
-}: Partial<PrivacyRequestParams>) => {
+  verbose,
+}: Partial<PrivacyRequestParams>): any {
```

Comment:
The Typescript compiler was struggling to resolve the return type on this function so I updated it to this to make it happy.